lib/use_cases/administrator/check_if_valid_ip.rb

```diff
@@ -0,0 +1,26 @@
+class CheckIfValidIp
+  def execute(address)
+    { success: address_valid_for_radius?(address) }
+  end
+
+private
+
+  def address_valid_for_radius?(address)
+    return false if address.nil?
```

Comment:
Was wondering if this line is required?
Same with the one below actually, is a subnet a valid ipv4?